at-tanvo/Controllers/session_controller.rb

```diff
@@ -0,0 +1,72 @@
+require 'bcrypt'
+require './Model/employee.rb'
+class SessionController
+  def login
+    system('clear')
+    puts 'xxxxxxx Login xxxxxxx'
+    puts 'Username: '
+    username = gets.chomp
+    puts 'Password:'
+    password = gets.chomp
+    result = Employee.new.check_user(username)
+    unless result.count == 0
+      result.each do |n|
+        next unless BCrypt::Password.new(n.values_at('password').first) == password
+        user_login =  Employee.new
+        user_login.id = x['id']
+        user_login.username = x['username']
+        return user_login
+      end
+    end
+  end
+
+  def register
+    name = ''
+    username = ''
+    loop do
+      system('clear')
+      puts 'xxxxxxx Register xxxxxxx'
+      puts 'Name : '
+      name = gets.chomp
+      puts 'Username : '
+      username = gets.chomp
+      result = Employee.new.check_user(username)
+      break if result.count == 0
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/aaa8a77d0d40b4c4a78140482c3fa32e36501a75/at-tanvo/Controllers/session_controller.rb#L34"></span>Style/NumericPredicate: Use <code>result.count.zero?</code> instead of <code>result.count == 0</code>. (https://github.com/rubocop-hq/ruby-style-guide#predicate-methods)</td>
    </tr>
  </tbody>
</table>